test/scripts/test_gen_vimdoc.py

```diff
@@ -0,0 +1,245 @@
+import os
```

Comment:
is this call by CI? I found it pretty easy to avoid regressions in `gen_vimdoc.py` simply by looking at the diff of generated files after running it.